docs/backend/algolia.md

```diff
@@ -27,15 +27,22 @@ application.
 
    ![algolia-up-3](https://user-images.githubusercontent.com/22895284/51078746-ad602c00-16ba-11e9-9927-d790ce03761e.png)
 
-6. You are all set up now! You can go to your dashboard.
+6. Sign up complete! You can go to your dashboard now.
 
    ![algolia-up-4](https://user-images.githubusercontent.com/22895284/51078747-ad602c00-16ba-11e9-8654-67c4d0f2e651.png)
 
-7. You can skip the tutorial, by clicking on API button.
+7. To avoid the trial period ending unexpectedly, you'll want to select a free
```

Comment:
```suggestion
7. To avoid the trial period ending unexpectedly, you'll want to select a community
```

to me, "free" implies "trial". could just be explicit and say that they _must_ be on a community plan here, i think?

Reply:
Whoops. Yup Merged this before I saw the suggestion. Not used to the suggested change UI. 😅😭